stable/gocd/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 1.3.3
+
+* [0c5eadd](https://github.com/kubernetes/charts/commit/0c5eadd): hostAliases for agents too (previously only on server)
```

Comment:
can we remove `(previously only on server)`? as we have the link to the commit, we can always get the commit message.

Reply:
Yes, fixed now. Thanks. 😊